ionic-app/components/pages/Lists.jsx

```diff
@@ -0,0 +1,52 @@
+import Store from '../../store';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/toost-net/toost-cloud/pullRequest?prid=8079923)
```suggestion
import Store from "../../store";
```